### DISCUSSION TITLE
Works extremely well.

### DISCUSSION THREAD

**Discussion by rjkenefick** [alobaili/camera-picker#2] [General]
No issues just wanted to comment on your work , this is turning out to be really good tons of people are looking for a swiftUI Camera Library that allows for recording video and saving to the phones photos library .  You seem to be pretty close keep it up !

**Comment by alobaili** [comment]
Thank you for the warm support. I plan to make this library very elegant and provide APIs for common use cases, as well as explore adding more advanced use cases that are not supported by `UIImagePickerController`. I expect sooner or later SwiftUI will have built-in Camera APIs, but this package will hopefully still be helpful for developers that want to support older OS versions.

I just enabled the Discussions tab in this repo. I will convert this issue to a discussion. Please feel free to keep the discussions there 🙌

**Comment by alobaili** [comment]
@rjkenefick Hey. I just wanted to let you know that I have added a `save()` function to save the picked item to the photos library.
You give it a try be adding version 0.1.1 of the package.
